src/years.txt

```diff
@@ -0,0 +1,85 @@
+1995: #########
+1957: #######
+2003: #######
+2000: ######
+2001: ######
+2014: ######
+2009: ######
+1994: #####
+1997: #####
+2007: #####
+1975: #####
+1984: #####
+1998: #####
+1999: #####
+2006: #####
+2011: #####
+2002: #####
+2004: #####
+2010: #####
+1993: ####
+1986: ####
+1959: ####
+1954: ####
+1980: ####
+2016: ####
+2017: ####
+1988: ####
+1979: ####
+2008: ####
+2012: ###
+1950: ###
+2005: ###
+2013: ###
+1940: ###
+2015/I: ###
+1991: ###
+1996: ###
+1982: ###
+1966: ###
+1939: ###
+1985: ###
+1976: ###
+1958: ##
+1961: ##
+1931: ##
+1962: ##
+1941: ##
+1948: ##
+1973: ##
+1981: ##
+1983: ##
+1987: ##
+1974: ##
+1968: ##
+1952: ##
+1927: ##
+1977: ##
+1946: ##
+1953: ##
+1960: ##
+1989: ##
+1992: ##
+1963: ##
+1955: #
+1925: #
+1921: #
+1926: #
+2015: #
+1965: #
+1942: #
+2013/I: #
```

Comment:
unclear year for me

Reply:
2013 year and I decade, this format provided by source file